creator/ingest_runs/tasks.py

```diff
@@ -1,13 +1,109 @@
-def run_ingest(ingest_run_id):
-    # TODO
-    print(f"Preparing ingest run {ingest_run_id}")
+import django_rq
+import logging
 
+from django.core.exceptions import ValidationError
 
-def cancel_ingest(ingest_run_id):
-    # TODO
-    print(f"Cancelling ingest run {ingest_run_id}")
+from creator.decorators import task
+from creator.files.models import FileType
+from creator.ingest_runs.models import IngestRun
+from creator.utils import stop_job
+
+
+logger = logging.getLogger(__name__)
+
+
+@task("run_ingest")
+def run_ingest(ingest_run_id=None):
```

Comment:
In all of our calls to `run_ingest` and `cancel_ingest` we pass in `str(ingest_run.id)`. Maybe we should just pass the uuid in and then do the string conversion in `get_ingest_run` so that you don't have a bunch of calls to `str()`